temporian/proto/core.proto

```diff
@@ -86,6 +87,21 @@ message Operator {
     message ListDType{
       repeated DType values = 1 [packed = true];
     }
+    message ListIndexKeys{
+      repeated IndexKey values = 1;
+
+      message IndexKey {
+        repeated IndexKeyItem values = 1;
+
+        message IndexKeyItem {
+          oneof types {
+            string str = 1;
```

Comment:
If we go with serializing the normalized form only, we don't need the "str" case.